addons/l10n_hr_kuna/models/template_hr_kuna.py

```diff
@@ -10,7 +10,7 @@ class AccountChartTemplate(models.AbstractModel):
     def _get_hr_kuna_template_data(self):
         return {
             'name': 'RRIF-ov računski plan za poduzetnike',
-            'code_digits': '0',
+            'code_digits': '6',
```

Comment:
Commit message also mentions LT but it is not in the diff

Reply:
Right. Thanks!